.travis.yml

```diff
@@ -1,19 +1,46 @@
-language: "python"
-python:
-   - "2.7"
-   - "3.5"
 sudo: false
-cache: pip
+dist: "trusty"
+language: "python"
+
+cache:
+  pip: true
+  apt: true
+
+addons:
+    apt:
+        sources:
+            ubuntu-toolchain-r-test
+matrix:
+    fast_finish: true
+    include:
+        - os: linux
+          python: 2.7
+          env: TF_VERSION=0.12.1 TF_TYPE=release
+        - os: linux
+          python: 3.5
+          env: TF_VERSION=0.12.1 TF_TYPE=release
+        - os: linux
+          python: 2.7
+          env: TF_VERSION=0.12.1 TF_TYPE=nightly
+        - os: linux
+          python: 3.5
+          env: TF_VERSION=0.12.1 TF_TYPE=nightly
+    allow_failures:
+        - env: TF_VERSION=0.12.1 TF_TYPE=nightly
 
 install:
-  - pip install flake8
+    - echo $TRAVIS_PYTHON_VERSION
+    - pip install -U flake8 opencv-python
+    - if [ -e requirements.txt ]; then pip install -r requirements.txt;else pip install -r requirements.pip;fi
+    - ./tests/install-tensorflow.sh
 
 before_script:
-  - flake8 --version
+    - flake8 --version
 
 script:
-  - flake8 .
-  - cd examples && flake8 .
```

Comment:
Yes but I ignored examples in /tox.ini, and examples/tox.ini is different from tox.ini (ignore certain type of errors).